packages/react-core/src/next/components/Select/SelectGroup.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import { MenuGroupProps, MenuGroup } from '../../../components/Menu';
+
+export interface SelectGroupProps extends Omit<MenuGroupProps, 'ref'> {
+  /** Anything which can be rendered in a select group */
+  children: React.ReactNode;
+  /** Classes applied to root element of select group */
+  className?: string;
+  /** Label of the select group */
+  label?: string;
+}
+
+export const SelectGroup: React.FunctionComponent<SelectGroupProps> = ({
+  children,
+  className,
+  label,
+  ...props
+}: SelectGroupProps) => (
+  <MenuGroup className={css(className)} label={label} {...props}>
```

Comment:
Yes it is possible, as all of these select components extend the menu components. In this case the `labelHeadingLevel` prop from MenuGroup would be used to set the heading level.

Perhaps the larger question is how we want to organize these extended props. I'm not a huge fan of duplicating the whole prop list into each Select component in case anything gets updated in Menu later on. Maybe in the docs I can pull in the Menu props as well as the Select props? @tlabaj

Reply:
Trying locally, it doesn't find the Menu components during tsDocgen because the next components are in a separate folder (so it's trying to parse the Menu props as menu-next files).

Reply:
@kmcfaul maybe we should open an issue in org so we can pull in props from src directory into next.  
AS far as exposing props for Menu.  I think if it is a common use case we can show it in the Select's prop table.  If it is something we are not really encouraging/ is not a common use case I would vote to not add it.

Reply:
I can open a quick follow up to go over the Menu props and see which ones we want to include for the dropdown/select wrappers & add them